packages/execution-environments/package.json

```diff
@@ -36,6 +36,7 @@
     "@metamask/post-message-stream": "^6.0.0",
     "@metamask/providers": "^9.0.0",
     "@metamask/snap-types": "^0.19.0",
+    "@metamask/snap-utils": "workspace:^",
```

Comment:
We're using `npm` instead of `yarn` to publish our packages. It'll probably break our publishing workflow.

It also breaks our linter it seems

Reply:
If you change it to `^0.19.0` it will still use the local version.